packages/data-driver-postgres/src/index.test.ts

```diff
@@ -304,90 +292,238 @@ test('nested m2o field', async () => {
 	expect(actualResult).toEqual(expectedResult);
 });
 
-test.todo('nested o2m field', async () => {
-	const collectionToJoin = randomIdentifier();
-	const joinField1 = randomIdentifier();
-	const joinField1Alias = randomIdentifier();
-	const joinField2 = randomIdentifier();
-	const primaryKeyField = randomIdentifier();
-	const collectionToJoinForeignKeyField = randomIdentifier();
+test('nested o2m field', async () => {
+	const localDesiredField = randomIdentifier();
+	const localDesiredFieldId = randomIdentifier();
+
+	const localPkField = randomIdentifier();
+	const localPkFieldId = randomIdentifier();
+
+	const foreignTable = randomIdentifier();
+	const foreignTableId = randomIdentifier();
```

Comment:
Why is there a `foreignTable` and a `foreignTableId`? Shouldn't this always be `foreignTable`?

Reply:
I guess your right 👍 I though we do `FROM {table} as {table_ID}` and hence should reference {table_ID} in the select node. but that's actually not how it is right now

Reply:
your're right. we don't do a join there and hence there is not actually no alias/id of the table

Reply:
got catch 👍